crates/algokit_transact_ffi/src/lib.rs

```diff
@@ -155,11 +243,11 @@ pub struct FeeParams {
 
 #[ffi_record]
 pub struct PaymentTransactionFields {
-    receiver: Address,
+    receiver: Account,
```

Comment:
I'm wondering if we make these strings?
I think it's worth putting the thought into getting the FFI layer supporting multisigs as well.

Reply:
I'll try this change with Strings. Since uniffi does not support unnamed fields in a struct, we'd need to have `Address` struct have `addr` field, resulting in an account having an address as `account.address.addr` which is not very ergonomic

Reply:
This should be addressed now